openapi/components.yml

```diff
@@ -0,0 +1,161 @@
+components:
+  schemas:
+    GeneralError:
+      type: object
+      properties:
+        title:
+          type: string
+          example: "Bad request"
+        type:
+          type: object
+          example: "about:blank"
+        status:
+          type: integer
+          format: int32
+          example: 400
+        detail:
+          type: string
+          example: "'type' is a required key"
+    Magic_Link:
+      description:
+        "Post JSON to request Magic link"
+      required:
+        - type
+        - to
+        - content
+      properties:
+        type:
+          type: string
+          example: "MAGIC_LINK"
+        to:
+          description: An email address.
+          type: string
+          format: email
+          pattern: '([A-Za-z0-9]+[.-_])*[A-Za-z0-9]+@[A-Za-z0-9-]+(\.[A-Z|a-z]{2,})+'
```

Comment:
Do we need the pattern as well as the format specification of `email`?

Reply:
Yeah I guess, after Gideon's comment I am  validating everything before it makes call to the API or hits functionality

Reply:
format email doesn't do all the checks so had to add pattern. Will go through with you in detail when huddling